packages/synapse-react-client/src/utils/functions/TableColumnSchemaUtils.test.ts

```diff
@@ -93,6 +144,38 @@ describe('TableColumnSchemaUtils', () => {
         newColumnId: NEW_COLUMN_MODEL_ID,
       })
     })
+
+    test('adding a new column that has an existing ID causes a thrown error', async () => {
```

Comment:
Add a test case to handle one of the current bugs -- no new columns should have IDs